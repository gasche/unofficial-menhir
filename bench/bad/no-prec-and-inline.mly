%start<int> a
%left foo
%%

a: {}

%inline b: %prec foo {} 
