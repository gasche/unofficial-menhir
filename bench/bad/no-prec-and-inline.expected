File "no-prec-and-inline.mly", line 7, characters 17-20:
Error: use of %prec is forbidden in an %inlined nonterminal definition.
